src/SprykerEco/Zed/ComputopShipment/Business/ComputopShipmentException.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\ComputopShipment\Business;
```

Comment:
usually we put exceptions under the Exception folder in order to keep root folders clean